pkg/cmd/releasenotes/render.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2021 ScyllaDB
+
+package releasenotes
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"regexp"
+	"strings"
+)
+
+const releaseNotesTemplate = `
+# Release notes for {{ .Release }}
+## Container images
+
+` + "```\ndocker.io/scylladb/scylla-operator:{{ .Release }}\n```" + `
+
+## Changes By Kind
+
+{{- range $section := .Sections }}
+  {{- if $section.PullRequests }}
+
+### {{ $section.Title }}
+
+    {{- range $pr := $section.PullRequests }}
+{{ FormatPR $pr }}
+    {{- end }}
+  {{- end }}
+{{- end }}
+
+---
+`
+
+var branchPrefixRe = regexp.MustCompile(`^(\[[^]].*])?(.*)`)
```

Comment:
Interesting that this works, I'd say it's missing escapes but maybe there is some rule saving it
```suggestion
var branchPrefixRe = regexp.MustCompile(`^(\[[^\]]+\])?(.*)`)
```

Reply:
Goland reports it's rendudant escaping, and tests proves that it works fine without `\`

Reply:
what about the rest? you are trying to match anything that isn't `]` but I read your regex as: `[` + a char that isn't `]` + any char + `]` - so why the `.*`?

Reply:
Right iteration is better, this is fixed in the updated version. I was reffering to escaping `]`.

Reply:
oh, sorry, I was only responding to the comments in mail, should have checked the updated version